src/Modules/SimplCommerce.Module.Comments/Views/Comment/List.cshtml

```diff
@@ -73,7 +74,10 @@
                         <div>
                             <button class="btn-add-review-reply btn btn-link">@Localizer["Add Reply"]</button>
                             <div class="add-review-reply d-none">
-                                @Html.Partial("/Modules/SimplCommerce.Module.Comments/Views/Comment/_ReplyForm.cshtml", new CommentForm { ParentId = comment.Id, EntityTypeId = Model.EntityTypeId, EntityId = Model.EntityId })
+                                @{ 
```

Comment:
Why you didn't pass the model as it was before?

Reply:
Yes I could have done this.